sdk/reya_websocket/resources/wallet.py

```
@@ -392,3 +404,31 @@ def subscribe(self, batched: bool = False) -> None:
     def unsubscribe(self) -> None:
         """Unsubscribe from wallet balances."""
         self.socket.send_unsubscribe(channel=self.path)
+
+
+class WalletAccountsResource(SubscribableParameterizedResource):
+    """Resource for wallet account-discovery updates."""
+
+    def __init__(self, socket: "ReyaSocket"):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)